### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by LumberKnot**
Created assert helper test

### CODE REVIEW

File: functions/test/utils/AssertHelper.test.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import { notNull } from "../../src/utils/AssertHelper";
+import { expect } from 'chai';
+
+describe('notNull tests', () => {
+    it('notNull(null)', () => {
```
Comment (Studsministern) [inline functions/test/utils/AssertHelper.test.ts]: I would recommend that we follow some semantics for the unit tests we add. There are normally two options: `it` or `test` - that do the same thing.

`it` should preferably be followed by "should"
`test` should prefereably be followed by "given/when/then"

https://markus.oberlehner.net/blog/naming-your-unit-tests-it-should-vs-given-when-then/

File: functions/test/utils/AssertHelper.test.ts [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        expect(() => notNull(null)).to.throw();
```
Comment (Studsministern) [inline functions/test/utils/AssertHelper.test.ts:6]: lambda function not needed here

**Review by Studsministern — CHANGES_REQUESTED** [review]
Maybe good to also include a command for how the coverage is achieved? Adding `nyc` for example?

**Review by Studsministern — APPROVED** [review]
LGTM!

File: functions/README.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+To test the functions, you can use the following command:
+
+```bash
+npm run test
+```
+
+To run coverage, you can use the following command:
+
+```bash
+npm run coverage -- <file_pattern>
```
Comment (Alce002) [inline functions/README.md]: Should be: npm run test:coverage

**Pull request merged**